dockstore-webservice/pom.xml

```diff
@@ -1232,6 +1232,7 @@
                     <generateModelTests>false</generateModelTests>
                     <generateModelDocumentation>false</generateModelDocumentation>
                     <generateSupportingFiles>true</generateSupportingFiles>
+                    <modelNameSuffix>v20beta</modelNameSuffix>
```

Comment:
Added this to avoid clashing class names between v2 final and v2 beta (which messes up the generated code badly)